manifests/configmap.yaml

```diff
@@ -36,7 +36,10 @@ data:
   team_admin_role: "admin"
   teams_api_url: http://fake-teams-api.default.svc.cluster.local
   workers: "4"
-  enable_load_balancer: "true"
+  # turn on/off load balancers for all Postgres clusters managed by the operator
+  # LB settings in cluster manifests take priority over these settings
+  # enable_master_load_balancer: "true"
```

Comment:
I still prefer these options being uncommented here with values 'false', what do you think?